### ISSUE TITLE
Error when downloading Firefox Focus or Firefox Klar

### ISSUE THREAD

**Issue by MrChocolatine** [Tobi823/ffupdater#79]
## Issue

Hello.

I have the exact same issue as => https://github.com/Tobi823/ffupdater/issues/63 .

I recently reinstalled my phone (details below) and now, if I try to install Firefox Klar or Firefox Focus I get the following error (same if I install FFUpdater through F-Droid or by downloading the APK from GitHub). (I never had this issue before reinstalling my phone)

```
Download failed

https://...

Application installation was not successful

You can now close this window
```

On the other hand, installing Iceraven, Firefox Lockwise, Firefox etc works fine.

I don't have any other clue on what is going on...

I do have access to Internet, phone freshly reinstalled, FFUpdater has the permission to install unknown apps...

Thank you for your help.

## Details

- Phone: Google Pixel 4a 5G
- OS: CalyxOS, version 2.9.0 (latest version available to this day)
- Version of Android: 11
- Version of FFUpdated: 74.4.0 (latest version available to this day)

Two days ago, before reinstalling my phone, I was using the exact same setup/phone/version and I did not have any issue with uninstalling/downloading/installing Firefox Klar/Focus.

**Comment by Nickoriginal** [comment]
Resolved by https://github.com/Tobi823/ffupdater/releases/tag/74.4.1

**Comment by Nickoriginal** [comment]
Should be (?) resolved by 74.4.1 (you cancheck it in the Releases), but I cannot check it.

**Comment by Tobi823** [comment]
Yes, I fixed it in 74.4.1 - but it's not released yet. (Creating a custom F-Droid repository for F-Droid + fixing/improving concurrent Kotlin coroutines have took a lot of time)

More details tomorrow

**Comment by MrChocolatine** [comment]
All right! I thought I was crazy when checking your the similar issue #63 and that I was doing something wrong, but apparently this is a new one. Thanks for the feedback!

**Comment by Tobi823** [comment]
It's now fixed in 74.4.3. 

You can either wait for the offical F-Droid release, install the APK file from GitHub/GitLab or try my personal F-Droid repository (see https://github.com/Tobi823/ffupdater#f-droid-repository).

I hope I don't upset the GitHub gods by using a GitHub repository as an F-Droid repository :smile:
